app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widgetHeader/widgetHeader.jsx

```diff
@@ -18,8 +18,10 @@ import ShareIcon from 'common/img/share-icon-inline.svg';
 import { widgetTypesMessages } from 'pages/inside/dashboardItemPage/modals/common/widgets';
 import { DescriptionTooltipIcon } from './descriptionTooltipIcon';
 import styles from './widgetHeader.scss';
+import widgetStyles from '../widget.scss';
 
 const cx = classNames.bind(styles);
+const cxWidget = classNames.bind(widgetStyles);
```

Comment:
I think it can be solved by using `customClass` property.
You can define hover behavior in `widget.scss`, and pass custom class to `WidgetHeader`

Reply:
Fixed.